plugins/kubernetes/app/models/concerns/kubernetes/deploy_group_permitted_params.rb

```diff
@@ -0,0 +1,15 @@
+module Kubernetes::DeployGroupPermittedParams
```

Comment:
@henders, I was not sure about this concern, but I think we still need it because we need to resolve which parameters are required and permitted and only then we can append the `_destroy` flag to the parameters. I couldn't find a better way to do this in the `deploy_group_permitted_params` callback bellow.

Well, I could have defined yet another event to append this flag to the parameters list, after resolving the required and permitted parameters. That would avoid the need of the decorator and the concern...